CMakeLists.txt

```diff
@@ -207,8 +227,9 @@ set(IREE_EXTERNAL_ROCM_HAL_DRIVER_REGISTER "iree_hal_rocm_driver_module_register
 
 #-------------------------------------------------------------------------------
 # Compiler Target Options
-# By default, all compiler targets supported by the current platform which do
-# not require external deps are enabled by default. This can be changed with:
+# By default, all compiler targets supported by the current platform are
+# enabled by default. Some compiler targets like CUDA will install external
+# deps as needed at configure time. This can be changed with:
```

Comment:
```suggestion
# By default, all compiler targets supported by the current platform are
# enabled. Some compiler targets like CUDA will install external deps as
# needed at configure time. This can be changed with:
```